doc/package.md

```diff
@@ -0,0 +1,176 @@
+# Package your application
+
+The `package` goal offers the following capabilities:
+
+* Provision an EAP server using Galleon.
+* Execute JBoss CLI commands and scripts to fine tune the provisioned server configuration.
+* Copy some extra content (e.g.: keystore, properties files) to the provisioned server installation.
+* Deploy your application in the provisioned server.
+
+By default the server is provisioned in the ``<project directory>/target/server`` directory.
+
+## Package a JAXRS application
```

Comment:
Do we need to document what happens if there are no layers in the configuration?

Reply:
I am not sure, in a cloud context, provisioning a bare-metal server with all default configs is not usable. The cloud fp defines a default-cloud-config layer to provision the correct default config with all modules. So layers should be always used.